Quick-Start [DA]/MVP/MVP+Makefile.md

```diff
@@ -1 +1,274 @@
+# Minimum Viable Product:
+A basic tutorial on creating a PostgreSQL extension using the bare minimum files and a Makefile:
+
+- A **control file**, which tells PostgreSQL some basic information about the extension, such as its name, version, and schema.
+- A **SQL script file**, which contains the SQL commands to create the extension's objects.
+- A **Makefile** is an optional but highly recommended file that helps automate the build process of the extension. It defines the compilation and installation steps, making it easier to build and install the extension on different systems.
```

Comment:
The Makefile is technically optional, but I don't think I'd mention that.

Reply:
Sure i'll change it.